proto/buf/alpha/registry/v1alpha1/organization.proto

```diff
@@ -66,6 +72,10 @@ service OrganizationService {
   rpc UpdateOrganizationMember(UpdateOrganizationMemberRequest) returns (UpdateOrganizationMemberResponse);
   // RemoveOrganizationMember remove the role of an user in the organization.
   rpc RemoveOrganizationMember(RemoveOrganizationMemberRequest) returns (RemoveOrganizationMemberResponse);
+  // GetOrganizationSettings gets the setting of a organization, including organization base roles.
+  rpc GetOrganizationSettings(GetOrganizationSettingsRequest) returns (GetOrganizationSettingsResponse);
+  // UpdateOrganizationBaseRoles update the organization base repository role, plugin role, and template role.
+  rpc UpdateOrganizationBaseRoles(UpdateOrganizationBaseRolesRequest) returns (UpdateOrganizationBaseRolesResponse);
```

Comment:
This is part of a larger conversation, should we optimize the API to reduce bloat?

Couldn't we send the desired organization settings instead of building out endpoints for each individual attribute. E.g.,

```
UpdateOrganization
UpdateOrganizationSettings

vs.

UpdateOrganizationBaseRoles
UpdateOrganizationName
UpdateOrganizationLogo
UpdateOrganizationSSOConfig
UpdateOrganizationBanner
UpdateOrganizationDescription
.... etc.
```

It seems like we can have fewer endpoints that do more, instead of endpoint per attribute. I'm pretty sure @robbertvanginkel had this same concern.

Reply:
One concern is how this works for partial updates - what happens when you just want to update the repository base role?

Reply:
That's an excellent point, re. updates. Potential solutions could be:

1. Send the entire payload. Most of the time, esp. on the frontend, you'll already have the organization context, so update the required fields and send the entire state. Similar to how you'd send k8s the desired state.
2. Do a bit more work to differentiate between null vs empty values. E.g., using `google.protobuf.StringValue` .. I don't think we utilize this pattern, but I've seen this work okay-ish. Build up an `UPDATE` query from the non-null fields.

These come to mind, maybe there are others?

Reply:
Another option is to use a FieldMask to define a field to update. Unfortunately I think all these suggestions are worse than just having multiple RPCs, IMO.

Reply:
For sending the entire payload, my concern is if there are multiple organization admin editing different parts of setting in the same time, which is very likely to be happened, so with in these 2 suggestions I will prefer to do something like (2) to only update non-null struct in the settings, however I don't think it can fully solve the scenario of updating only the repository base role, we still need to pass all 3 base roles anyway

Reply:
Partial updates aren't awful, they do require a bit more logic in the business layer but the tradeoff is a cleaner and potentially easier API to work with, IMO.

Probably beyond the scope of this PR, but something that worries me is the API surface we'll end up with as we continue to add functionality.

Reply:
I guess it still depends on how small we divide them to multiple RPCs. For me, grouping the update of 3 base roles together is in the balance that acceptable in both API bloat concern and RPC boundary concern. However, if we continue to add functionality, it's true that it will still end up with so many update endpoints.

It's beyond the scope of this PR, but I guess it's a good timing to discuss about this.

Reply:
Considering a GUI in this form:

```
  Organization Base roles
------------------------------

  Repositories: [ Write ▼ ]

  Plugins:      [ Read  ▼ ]

  Templates:    [ Read  ▼ ]

                   [ Save ]
```

Would the client be expected to invoke three RPCs on Save? If the user changes only the repository base role and hits Save, should only the repository base role be transmitted?

A single RPC with partial updates could give us flexibility for different (or changing) use cases. However, I personally think that it might be too early for this abstraction.